docs/product-installation-guide/src/main/asciidoc/eap-business-central-install-proc.adoc

```diff
@@ -8,16 +8,16 @@ Business Central is a web console that enables you to perform the following task
 * Create new tasks and notifications.
 
 .Prerequisites
-* {EAP} 7.0.7 installed on your hard drive. The base directory of the EAP installation is referred to as `__EAP_HOME__`.
+* {EAP} 7.0.8 installed on your hard drive. The base directory of the EAP installation is referred to as `__EAP_HOME__`.
 * The following file, available from the site that contains this document:
 +
-ifdef::BPMS[`jboss-bpmsuite-7.0.0.LA.ER3-collection.zip`]
-ifdef::BRMS[`jboss-brms-7.0.0.LA.ER3-collection.zip`]
+ifdef::BPMS[`jboss-bpmsuite-7.0.0.LA-add-ons.zip.zip`]
+ifdef::BRMS[`jboss-brms-7.0.0.LA-add-ons.zip`]
 
 .Procedure
-. Extract the `jboss-bpmsuite-7.0.0.LA.ER3-collection.zip` archive to a temporary directory. In the following examples this directory is called `__TEMP_DIR__`.
-. Extract the `jboss-bpmsuite-7.0-business-central-eap7.zip` file, available from the `jboss-bpmsuite-7.0.0.LA.ER3-collection` directory that you extracted in the previous step.
-. Copy the contents of the `__TEMP_DIR__/jboss-bpmsuite-7.0.0.LA.ER3-collection/jboss-bpmsuite-7.0-business-central-eap7/jboss-eap-7.0` directory to `__EAP_HOME__`. When asked to overwrite files or merge directories, select *Yes*.
+. Extract the `jboss-bpmsuite-7.0.0.LA-add-ons.zip.zip` archive to a temporary directory. In the following examples this directory is called `__TEMP_DIR__`.
+. Extract the `jboss-bpmsuite-7.0.0.LA-business-central-eap7.zip` file, available from the `jboss-bpmsuite-7.0.0.LA-add-ons.zip` directory that you extracted in the previous step.
```

Comment:
Might read better as follows, but no biggie:
Extract the `jboss-bpmsuite-7.0.0.LA-business-central-eap7.zip` file from the `jboss-bpmsuite-7.0.0.LA-add-ons.zip` directory that you extracted in the previous step.